index.html

```diff
@@ -0,0 +1,612 @@
+                <p class="popular-paragraph"><a href="#">Coronavirus Live Updates: Toll Rises as Emergency Declaration Is Considered</a></p>
+                <hr class="popular-line">
+                <p class="popular-paragraph"><a href="#">Republicans Move to Block Impeachment Witnesses, Driving Toward Acquittal</a></p>
+                <hr class="popular-line">
+                <p class="popular-paragraph"><a href="#">Mike Pompeo’s NPR Interview Left a Lot to Be Considered, Seth Meyers Says</a></p>
+            </div>
+        </div>
+        <div class="clear1"></div>
+        <hr class="editors-picks-line">
+        <div class="editors-picks-container">
+          <h3><a class="editors-title-link" href="#">Editor's Picks</a></h3>
+            <figure class="editors-picks">
+                <img alt="Saturday night" src="https://static01.nyt.com/images/2020/01/26/arts/26snl-2/26snl-2-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale"> 
+                <figcaption class="editor-outer-figcaptions">
+                  <a class="photo-by-links" href="#"><section class="photo-by"><h5>NBC</h5><br /></section></a>
+                    <a href="#" class="figcap-link">'Saturday Night Live' Spoofs Trump's Impeachment <span class="editor-time"><br /><time datetime="2008-02-14 20:00">Jan. 26</time></span></a>
+                </figcaption>
+            </figure>
+            <figure class="editors-picks">
+                <img alt="Quit" src="https://static01.nyt.com/images/2020/01/21/fashion/21quit-promo-condom/21quit-promo-14-square640.jpg?quality=75&auto=webp&disable=upscale&width=350">
+                <figcaption class="editor-outer-figcaptions">
+                  <a class="photo-by-links" href=""> <section class="photo-by"><h5>Photo Illustration by Tony Cenicola/The New York times</h5><br /></section></a>
+                    <a href="#" class="figcap-link">I Quit Sex for 12 years <span class="editor-time"><br /><time datetime="2008-02-14 20:00">Jan. 20</time></span></a>
+                </figcaption>
+            </figure>
+            <figure class="editors-picks">
+                <img alt="alamos" src="https://static01.nyt.com/images/2020/01/28/science/00NEWSPY1/merlin_166359192_36d3b52f-65fa-4a73-99ca-c9560fc52f97-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale">
+                <figcaption class="editor-outer-figcaptions">
+                  <a class="photo-by-links" href=""><section class="photo-by"><h5>Los Alamos National Laboratory</h5><br /></section></a>
+                    <a href="#" class="figcap-link">Fourth Spy at Alamos Knew A-Bomb's Inner Secrets <span class="editor-time"><br /><time datetime="2008-02-14 20:00">Jan. 27</time> </span></a>
+                </figcaption>
+            </figure>
+        </div>
+        <hr class="double-line">
+        <div class="bottom-links-container">
+          <svg class="footer-logo" height="25" width="185" xmlns="http://www.w3.org/2000/svg">
```

Comment:
- [ ]  put height and width selectors in a class or Id in your CSS file as no inline styling is permitted in the HTML